test/yargs.js

```diff
@@ -324,6 +348,22 @@ describe('yargs dsl tests', function () {
       r.logs.join(' ').should.match(/Choose yer command:/)
     })
 
+    it('handles a missing locale', function () {
+      loadLocale('zz_ZZ.UTF-8')
+
+      var r = checkOutput(function () {
+        yargs(['snuh', '-h'])
+          .command('blerg', 'blerg command')
+          .help('h')
+          .wrap(null)
+          .argv
+      })
+
+      yargs.locale().should.equal('zz')
```

Comment:
oh, was just pointing out that locales tend to look like this `pt_PT.UTF-8` rather than `pt_PT`, but I don't think we'd ever have a locale file called `pt_PT.UTF-8`.